operator/v1/types_ingress.go

```diff
@@ -384,8 +384,32 @@ type LoadBalancerStrategy struct {
 	//
 	// +optional
 	ProviderParameters *ProviderLoadBalancerParameters `json:"providerParameters,omitempty"`
+
+	// dnsManagementPolicy indicates if the lifecyle of the wildcard DNS record
+	// associated with the load balancer service will be managed by
+	// the ingress operator.
+	//
+	// +kubebuilder:default:="Managed"
```

Comment:
Before this field was added, was the default logically Managed?

Reply:
Yes, prior to this new field, the behavior is synonymous to `Managed`.